flight/targets/pipxtreme/fw/pios_config.h

```diff
@@ -39,8 +39,9 @@
 #define PIOS_INCLUDE_OPENLRS_RCVR
 
 /* Supported receiver interfaces */
-#define PIOS_INCLUDE_PPM
 #define PIOS_INCLUDE_GCSRCVR
+#define PIOS_INCLUDE_PPM
+#define PIOS_INCLUDE_SBUS
```

Comment:
What's special about S.BUS?

Reply:
Will be using it as one way of tx-to-module comms.